tensorflow_addons/losses/sparsemax_loss_test.py

```diff
@@ -104,6 +104,33 @@ def test_gradient_against_estimate(dtype):
     test_utils.assert_allclose_according_to_type(jacob_sym, jacob_num)
 
 
+@pytest.mark.parametrize("dtype", ["float32", "float64"])
```

Comment:
`_tf_sparsemax_loss` is not a test. As such, the pytest decorator has no effect.

Reply:
Ops, my bad, updated.